iotdb/src/main/java/org/apache/iotdb/db/engine/filenode/FileNodeManager.java

```diff
@@ -694,7 +692,11 @@ public void endQuery(String deviceId, int token) throws FileNodeManagerException
     try {
       LOGGER.debug("Get the FileNodeProcessor: {} end query.",
           fileNodeProcessor.getProcessorName());
-      fileNodeProcessor.removeMultiPassLock(token);
+      fileNodeProcessor.decreaseMultiPassCount(token);
+    } catch (FileNodeProcessorException e) {
+      throw new FileNodeManagerException(String
+          .format("FileNodeProcessor of [%s] meets error when ending query",
```

Comment:
May be the the format of log should be consistent with other.
@jixuan1989 We should  give a log guides for our developer.

Reply:
fixed